kafka-rest/src/main/java/io/confluent/kafkarest/controllers/SchemaManagerImpl.java

```diff
@@ -109,12 +124,19 @@ private RegisteredSchema getSchemaFromSchemaId(
 
   private int getSchemaVersion(String subject, ParsedSchema schema) {
     try {
-      return schemaRegistryClient.getVersion(subject, schema);
+      return schemaRegistryClient.get().getVersion(subject, schema);
     } catch (IOException | RestClientException e) {
-      throw new SerializationException(
+      String schemaString;
+      if (schema.canonicalString().equals("\"int\"")) {
+        schemaString = "";
+      } else {
+        schemaString = schema.canonicalString();
+      }
```

Comment:
nit: I think this fits on a single line with a ternary op:
```
String schemaString = schema.canonicalString().equals("\"int\"") ? "" : schema.canonicalString();
```

Reply:
We should trust `google-java-format`.

Reply:
Are you saying `google-java-format` will break the proposed one-liner? I would expect it won't if it fits within our line length limits, which this does.

Reply:
Oh, sorry. I read too fast. I thought you just reformatted the code. Ignore me. It's Friday. :)